config/default/kustomization.yaml

```diff
@@ -1,6 +1,5 @@
 # Adds namespace to all resources.
-# namespace: hardware-classification-controller-system
-namespace: metal3
+namespace: hcc-system
```

Comment:
I guess "hcc" is "hardware classification controller"? Maybe we can just call the namespace hardware-classification?

Reply:
We will address this in PR #7